dev_guide/builds.adoc

```diff
@@ -105,6 +105,39 @@ The following options are specific to the
 link:../architecture/core_concepts/builds_and_image_streams.html#source-build[S2I
 build strategy].
 
+[[force-pull]]
+
+=== Force Pull
+
+By default, if the builder Docker image specified in the  build confirguration is available in OpenShift's local,
+internal registry, that image will be used.  However, to override the local image and refresh it from the Docker image
```

Comment:
same comment as below... the hierarchy here is "copy of the image stored on the node (previously docker pulled to the node)" followed by "the current/up to date version of the image stored in some registry (could be the openshift internal registry, could be dockerhub, could be some other registry the customer referenced as the location for their image)